src/ros_filter.cpp

```diff
@@ -1793,6 +1804,15 @@ void RosFilter<T>::initialize()
 template<typename T>
 void RosFilter<T>::periodicUpdate()
 {
+  // Wait for the filter to be enabled
+  if (!enabled_)
+  {
+    RCLCPP_INFO_ONCE(this->get_logger(),
```

Comment:
Yeah, those are in [Dashing](http://docs.ros2.org/dashing/api/rclcpp/logging_8hpp.html). In [Eloquent](http://docs.ros2.org/latest/api/rclcpp/logging_8hpp.html) there is also `*_THROTTLE` and `*_STREAM`. I thought of simply porting to Eloquent so I didn't have to reformat all the `_STREAM` calls, but just stuck with Dashing... It would have been easier to port to Eloquent.

Reply:
WE GOT `THROTTLE` TOO?!

You're speaking my language